numba/tests/test_array_methods.py

```diff
@@ -1780,6 +1780,35 @@ def check(a, b, expected):
         check(arr, arr.T, False)
         check(arr, arr[:-1], False)
 
+    def test_record_is(self):
+        def check(a, b, expected):
+            pyfunc = identity_usecase
+            cfunc=njit(pyfunc)
+
+
+            result = cfunc(a, b)
+            self.assertEqual(result, expected)
+
+        record_type = np.dtype([("a", "i4"), ("b", "i4")])
+
+        # Test case: Same records
+        a = np.zeros(1, dtype=record_type)[0]
+        b = a.copy()
+        check(a, b, True)
+
+        # Test case: Different records
+        c = np.ones(1, dtype=record_type)[0]
+        check(a, c, False)
+
+        # Test case: Same records with different values
+        d = np.zeros(1, dtype=record_type)[0]
+        d['a'] = 42
+        check(a, d, False)
```

Comment:
Rather than give the expected result as an argument, compute it by calling `pyfunc(a, b)`. This should fix the CI issue.

```suggestion
    def test_record_is(self):
        def check(a, b):
            pyfunc = identity_usecase
            cfunc = njit(pyfunc)
            result = cfunc(a, b)
            expected = pyfunc(a, b)
            self.assertEqual(result, expected)

        record_type = np.dtype([("a", "i4"), ("b", "i4")])

        # Test case: Same records
        a = np.zeros(1, dtype=record_type)[0]
        b = a.copy()
        check(a, b)

        # Test case: Different records
        c = np.ones(1, dtype=record_type)[0]
        check(a, c)

        # Test case: Same records with different values
        d = np.zeros(1, dtype=record_type)[0]
        d['a'] = 42
        check(a, d)
```